packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -138,6 +194,60 @@ class CodePushClient {
         .toList();
   }
 
+  /// List all channels for the provided [appId].
+  Future<List<Channel>> getChannels({required String appId}) async {
+    final response = await _httpClient.get(
+      Uri.parse('$hostedUri/api/v1/channels').replace(
```

Comment:
The version probably doesn't belong in every single caller?  (A later problem to fix.)

Reply:
yeah agreed we can extract that in a subsequent PR 👍